Calling/src/app/views/HomeScreen.tsx

```diff
@@ -131,7 +131,8 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
 
   const isMobileSession = useIsMobile();
 
-  const showDisplayNameField = !teamsIdentityChosen;
+  let showDisplayNameField = true;
+  showDisplayNameField = !teamsIdentityChosen;
```

Comment:
We should revert this change for simplification

Reply:
This should have been a direct port from the calling sample in UI Lib so we may need to also simplify this statement there as well

Reply:
![image](https://github.com/user-attachments/assets/52f5cef1-ea39-4752-92bf-62456a1ee889)